client/src/Components/Layouts/Customers/index.jsx

```diff
@@ -1,20 +1,134 @@
+            }
+        } else this.setState({ customers: this.state.allData })
+    };
+
+
+    dateFilter = (value, customers) => {
+        if (value.length !== 0) {
+            const from = value[0]._d.setHours(0, 0, 0, 0);
+            const to = value[1]._d.setHours(0, 0, 0, 0);
+            let filterCustomer = customers.filter(customer => {
+                let createdDate = new Date(customer.dt_create_at).setHours(0, 0, 0, 0);
+                if (createdDate >= from && createdDate <= to) return customer;
+            })
+            this.setState({
+                customers: filterCustomer,
+                date: value,
+                filteredcustomersDate: filterCustomer
+
+            })
+        }
+        else {
+            this.setState({
+                customers: this.state.allData
+            })
+        }
+    }
+    nameFilter = (name, customers) => {
+        const value = name.trim();
+        let filterCustomerName = customers.filter(customer => {
+            if (customer.s_name.includes(value)) return customer;
+        })
+        this.setState({
+            customers: filterCustomerName,
+            name,
+            filteredcustomersName: filterCustomerName
+        })
     }
-    render(){
-        return(
-            <div>
-                <Sidebar/>
-                <Navbar name={this.state.name}/>
-                <Header  />
+    render() {
+        const { title } = this.state
+        return (
+            <div style={{ display: "flex" }}>
+                <Sidebar />
+                <div style={{ display: "flex", "flex-direction": "column", width: "100vw" }}>
+                    <Navbar />
+                    <Header title='إدارة المستخدمين' Icon={<Icon type="team" />} />
+                    <div className='addcustomer'>
+                        <Button name='إضافة مستخدم' icon={<Icon type="user" />} />
+                        <div className="filtercontainer">
+                            <div classNam="filtercontainer__orderdate">
+                                <RangePicker
+                                    showTime={{ format: 'HH:mm' }}
+                                    format="YYYY-MM-DD HH:mm"
+                                    placeholder={['من', 'الى']}
+                                    onChange={e => this.filterfunction(e, '', 'date')}
+                                    style={{ direction: "rtl" }}
+                                />
+                                <span className='filtercontainer__orderdate-date'>فلترة حسب الوقت</span>
+                            </div>
+                            <Input size="defaul" placeholder="فلترة حسب الاسم" className="filtercontainer__ordername" onChange={e => this.filterfunction('', e.target.value, 'name')} />
+                        </div>
+                    </div>
+                    <Table pageName="customers" columns={this.state.customers} classname='tablecustomer-container' viewPopup={viewPopup}
```

Comment:
the pop up need to update as figma, but it's okay now and you can open new PR and work on it